pyproject.toml

```diff
@@ -13,7 +13,7 @@ dagster-scaleway = "dagster_scaleway.cli:main"
 
 [tool.poetry.dependencies]
 python = "^3.11"
-scaleway = "^1.4.1"
+scaleway = "^2.0.0"
```

Comment:
note: do not merge as this, there's an issue with `list_job_definitions_all` with this version of the sdk

Reply:
fixed in release v2.1.0